apps/fhir/bluebutton/utils.py

```diff
@@ -909,3 +909,58 @@ def build_oauth_resource(request, format_type="json"):
         ]
 
     return security
+
+
+def strip_format_for_back_end(pass_params):
```

Comment:
Since these are moving around anyway can they be put in a file with a name matching their purpose?

Reply:
The actual answer is that these functions are unnecessary, and should just be removed. Why don't I go ahead and do that.